go.mod

```diff
@@ -17,16 +17,19 @@ require (
 	github.com/gofrs/uuid v4.4.0+incompatible
 	github.com/google/go-containerregistry v0.20.3
 	github.com/hashicorp/go-multierror v1.1.1
+	github.com/joho/godotenv v1.5.1
 	github.com/kairos-io/go-ukify v0.2.5
 	github.com/kairos-io/kairos-agent/v2 v2.16.4
 	github.com/kairos-io/kairos-sdk v0.7.3
 	github.com/kairos-io/netboot v0.0.0-20241104101831-1454e04fdb07
 	github.com/klauspost/compress v1.18.0
+	github.com/labstack/echo/v4 v4.13.3
 	github.com/mudler/go-processmanager v0.0.0-20240820160718-8b802d3ecf82
 	github.com/mudler/yip v1.15.0
 	github.com/onsi/ginkgo/v2 v2.23.0
 	github.com/onsi/gomega v1.36.2
 	github.com/otiai10/copy v1.14.1
+	github.com/robert-nix/ansihtml v1.0.1
```

Comment:
This project doesn't have many dependencies but hasn't got any commit for the last 3 years. We should keep an eye for possible vulnerabilities in any of the [deps](https://github.com/robert-nix/ansihtml/blob/master/go.sum).

Reply:
created an issue for this kairos-io/kairos#3267